docs/running_lbann.rst

```diff
@@ -1,96 +1,434 @@
 .. role:: bash(code)
           :language: bash
+.. role:: python(code)
+          :language: python
 
-====================
+============================================================
 Running LBANN
-====================
+============================================================
 
-The basic template for running LBANN is
+------------------------------------------------
+Anatomy of an LBANN experiment
+------------------------------------------------
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Parallelism
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+LBANN is run under the `MPI
+<https://en.wikipedia.org/wiki/Message_Passing_Interface>`_ paradigm,
+i.e. with multiple processes that communicate with message
+passing. These processes are subdivided into "trainers." Conceptually,
```

Comment:
I would say something more like `this set of processes is partitioned into "trainers".`.